bot/util.go

```diff
@@ -447,13 +447,20 @@ func CheckDiscordErrRetry(err error) bool {
 	return true
 }
 
-func IsNormalUserMessage(msg *discordgo.Message) bool {
+var allowedMsgTypes = []int{int(discordgo.MessageTypeDefault), int(discordgo.MessageTypeReply), int(discordgo.MessageTypeThreadStarterMessage)}
+
+// set strictType to true to only permit default, reply, and thread starter message types
+func IsNormalUserMessage(msg *discordgo.Message, ignoreMsgType bool) bool {
```

Comment:
What do you think about separating into two functions, `IsUserMessage` for the current check and `IsNormalUserMessage` for the behavior with `ignoreMsgType` set to `true`? The two are meaningfully different and combining both into one function with a flag is a little confusing. Distinct names is also clearer from the call site (instead of opaque true/false values.)